Jenkinsfile

```diff
@@ -49,41 +42,24 @@ pipeline {
             // so we can retrieve the version in later steps
             sh "echo \$(jx-release-version) > VERSION"
             sh "mvn versions:set -DnewVersion=\$(cat VERSION)"
-            sh "mvn versions:set-property -Dproperty=activiti.version -DnewVersion=\$(cat VERSION)"
+
+            sh "git add --all"
+            sh "git commit -m 'release \$(cat VERSION)' --allow-empty"
```

Comment:
Can we have capital for the fist letter of the commit message?

Reply:
@erdemedeiros good point.. not sure if that is because some kind of convention that github understands..

Reply:
@erdemedeiros we got that line from how JX is doing releases.. we might want to ask them before changing it